### PR TITLE
 Complete OpenCTI documentation

### PR DESCRIPTION
**Pull request by Lhorus6**
- Add documentation on technical dates vs. functional dates in OpenCTI.
- Review CSV feed ingestion documentation.
- Improve documentation on markings (order / marking hierarchy).

### CODE REVIEW

File: docs/administration/segregation.md [hunk 1]
```diff
@@ -22,63 +18,62 @@ In OpenCTI, data is segregated based on knowledge marking. The diagram provided
...
-![TLP diagram](assets/tlp-diagram.png)
+To create a marking, you must first possess the capability `Manage marking definitions`. For further information on user administration, please refer to the [Users and Role Based Access Control page](./users.md).
 
-### Create new markings
+Once you have access to the settings, navigate to "Settings > Security > Marking Definitions" to create a new marking.
 
-In order to create a marking, you must first have the ability to access the Settings tab. For example, a user who is in a group with the role of Administrator can bypass all capabilities or a user who is in a group with the role that has `Access administration` checked can access the Settings tab. For more details about user administration, please see the [Users and Role Based Access Control page](./users.md).
+A marking consists of the following attributes:
 
-![Authorize access administration](assets/authorize-access.gif)
```
Comment (EinatAR) [inline docs/administration/segregation.md:39]: I really like the gifs for showing the steps of a process!!!

Comment (Lhorus6) [reply docs/administration/segregation.md:39]: It's not from me. But yes, the gifs in the docs are great!

File: docs/administration/segregation.md [hunk 2]
```diff
@@ -22,63 +18,62 @@ In OpenCTI, data is segregated based on knowledge marking. The diagram provided
...
+- The user is a member of a group that has access to a marking of the same type, with an equal or higher hierarchical order.
 
-![Update allowed markings in a group](assets/update-group-marking.png)
+!!! note "Access to an object with several markings"
 
-### Default marking definitions
+    Access to all markings attached to an object is required in order to access it (not only one).
 
-To apply a default marking when creating a new entity or relationship, you can choose which marking to add by default from the list of allowed markings. You can add *only one marking per type*, but you can have multiple types.
+!!! tip "Automatically grant access to the new marking"
```
Comment (EinatAR) [inline docs/administration/segregation.md:53]: I didn't understand why it has a copy/paste option. 

I understand the topic and explanation.

Comment (EinatAR) [reply docs/administration/segregation.md:53]: image

Comment (Lhorus6) [reply docs/administration/segregation.md:53]: In terms of format/display ?

**Pull request merged**